src/lib.ts

```diff
@@ -121,3 +121,4 @@ export {
 
 export * from './nano_contracts/types';
 export * from './models/types';
+export type { IHathorWallet, OutputRequestObj, DataScriptOutputRequestObj } from './wallet/types';
```

Comment:
In `models/types` we have types with the same name as in the `wallet/types`, so I can't do `export * from './wallet/types';`, I get the following error doing it

```
src/lib.ts:124:1 - error TS2308: Module './models/types' has already exported a member named 'Authority'. Consider explicitly re-exporting to resolve the ambiguity.

124 export * from './wallet/types';
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/lib.ts:124:1 - error TS2308: Module './models/types' has already exported a member named 'Balance'. Consider explicitly re-exporting to resolve the ambiguity.

124 export * from './wallet/types';
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/lib.ts:124
```

We should normalize these types, so we can export all.